new_core/config/debugPages.js

```diff
@@ -2,7 +2,8 @@ export const MAIN_INFO = {
   name: "Performance",
   data: {
     row1: {
-      col1: [ "FPS", "performance.gameloop.fps" ],
+      col1: [ "TFPS", "performance.gameloop.tfps" ],
+      col2: [ "JFPS", "performance.gameloop.jfps"]
```

Comment:
what's the difference between tfps and jfps. I would suggest to use better property names here. Even FPS is not good as name, but at least well known.

Reply:
They both represent the same data, calculating frames per second

TFPS = Tapsi Frames Per Second, which gets the delta time each frame
JFPS = JSRulz Frame Per Second, which gets the frames each second

Both are useful for different reasons, but I definitely use jfps for impact code. I leave tfps, because it is useful for figuring out an average of fps over time. We've been back and forth on this (taking it out, putting it back in). Can a compromise be made, maybe?

Reply:
ok, I have a big smile on my face. :D

Reply:
but tfps is not the delta time. if I am not mistaken, then you mean it will be calculated every tick right ?

Reply:
Yeah, it'll be calculated each tick

Reply:
okay we should remove tfps , since we have delta as model variable. JFPS should be renamed to FPS then. I think you only want to see bad frames (high delta) and a flattened FPS.

Reply:
IMHO jfps is definitely better than the current FPS variable

Reply:
Yeah, but they both have their uses. Like, you can see which frames spike exactly with tfps. I learned to use both to get a good idea of the game performance without having to crack open an editor and look at two debug windows at once. So much information about profiling from just those two bits of information

Reply:
I'll remove it later on. Duplicates the delta value in a different format.